packages/atomic/src/components/common/facets/facet-search/facet-search-input.tsx

```diff
@@ -16,7 +16,7 @@ export const FacetSearchInput: FunctionalComponent<FacetSearchInputProps> = (
   props
 ) => {
   const label = props.i18n.t(props.label);
-  const search = props.i18n.t('search');
+  const search = props.i18n.t('facet-search-input');
```

Comment:
I didn't want to do this until v2 originally, but it's not a major breaking change so i guess its fine 😄

Reply:
I didn't see this in the list of v2 changes, did I miss it somewhere?